packages/docs/versioned_docs/version-2.0.0-beta-3/welcome.md

```diff
@@ -32,8 +32,11 @@ UniRep's key offering is providing developers with a pre-built, audited system t
 :::
 ### Application built by our core contributors ###
 
-- Built by [Kittybest](https://github.com/kittybest),  Web2 Authentication Bridging: [My-Badge](https://github.com/kittybest/my-badge)
-- Built by [CJ](https://github.com/CJ-Rose) ,  Craiglist reinvent: [Trustlist](https://github.com/CJ-Rose/trustlist)
+- Built by [Vimwitch](https://github.com/vimwitch),  Do things with ethereum addresses in zk: [ZKETH repo](https://github.com/Unirep/zketh) App [try it now](https://zketh.io/)
+- Built by [Vimwitch](https://github.com/vimwitch),  DAO: [ZKDAO repo](https://github.com/vimwitch/zkdao) 
+- Built by [Kittybest](https://github.com/kittybest),  Web2 Authentication Bridging: [My-Badge repo](https://github.com/kittybest/my-badge)
+- Built by [CJ](https://github.com/CJ-Rose) ,  Craiglist reinvent: [Trustlist repo](https://github.com/CJ-Rose/trustlist)
+- Built by [Vivian](https://github.com/vivianjeng) & [Kittybest](https://github.com/kittybest) , Social network with reputation touch: Unirep.social app[Unirep.social@Staging](https://social-staging.unirep.io/)
```

Comment:
Built by unirep team?
and it can be linked to https://github.com/unirep/unirep-social
since https://social-staging.unirep.io/ is not working now :(